helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml

```diff
@@ -105,6 +105,10 @@ spec:
     rules:
     - expr: sum(engine_daemon_image_actions_seconds_count) by (cluster_id, cluster_type, action)
       record: aggregation:docker:action_count
```

Comment:
@marians could you also add init containers here ? I think the metric is named kube_pod_init_container_info

Reply:
I am using the same here https://github.com/giantswarm/prometheus-rules/pull/959/

Reply:
Very good suggestion! These weren't on my radar as an extra metric. Will make an extra PR.